.circleci/tests/golden/default/wiz-sensor.golden.yaml

```diff
@@ -0,0 +1,327 @@
+---
```

Comment:
Sensor can't support golden files, it is auto merged from develop with a bot.
Let's start with ac/ k8s connector/ broker and then support more components with the relevant owner.